zulip/zulip/__init__.py

```diff
@@ -1591,6 +1591,18 @@ def render_message(self, request: Optional[Dict[str, Any]] = None) -> Dict[str,
             request=request,
         )
 
+    def get_message_read_receipts(self, message_id: int) -> Dict[str, Any]:
+        """
+        Example usage:
+
+        >>> client.get_message_read_receipts(42)
+        {'result': 'success', 'msg': '', 'user_ids': [42]}
+        """
+        return self.call_endpoint(
+            url=f"messages/{message_id}/read_receipts",
+            method="GET",
+        )
```

Comment:
Our usual convention is to use the same names as the URL suffices on https://zulip.com/api/get-read-receipts.

Reply:
so should I change the function name to get_read_receipts, not sure if that is what you meant

Reply:
Yes